src/main/features/response/views/statement-of-means/court-orders.njk

```diff
@@ -31,22 +28,23 @@
     <form method="post" novalidate
           class="analytics-submit-event-trigger"
           data-event-action="SoM: Are you paying any other court orders?"
-          data-event-label-from="hasAnyCourtOrders">
+          data-event-label-from="declared">
```

Comment:
I"m not sure this should be "declared". this will be shown in GA and should probably say something more meaningful.

Reply:
This won't be visible in GA. Action will be report as `SoM: Are you paying any other court orders?` and value will be taken from `declared` form field.

Reply:
The fact you pointed event made me review all events and make event name in sync with the question asked on the page. There were 4 places out of sync. Thanks.